src/plugins/intel_npu/src/compiler/include/custom_stream_buffer.hpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <iostream>
+#include <streambuf>
+#include <vector>
+
+namespace intel_npu::driverCompilerAdapter {
+
+/**
+ * @brief Custom stream buffer for reading and writing data of data larger than 2G on Windows
+ */
+class CustomStreamBuf : public std::streambuf {
+public:
+    CustomStreamBuf(size_t bufferSize = 1024) : _buffer(bufferSize), _readPos(0), _writePos(0) {
+        // Update read and write pointers
+        char* base = _buffer.data();
+        this->setg(base, base, base);
+        this->setp(base, base + _buffer.size());
+    }
+
+    std::streampos tellp() {
```

Comment:
missed this: tellp/tellg are not streambuf members. remove

Reply:
Removed, Thanks!